js/engine/Main.ml

```diff
@@ -71,22 +71,44 @@ let _ =
          | Some lang -> Js.some (Js.string (Lang.to_lowercase_alnum lang))
          | None -> Js.null
 
-       method execute language rule_file source_file : string =
+       method execute language rule_file root source_files : string =
```

Comment:
Maybe a comment explaining why we need root here? Just explain this is for semgrep-pro or something

Reply:
Also maybe add a comment note such as
(* note that here we take a list of source files! The playground generally parses just one file at a time, but
 * we accept a list of files because XXX
 *)

Reply:
Or maybe just say turbo mode used to be just for semgrep OSS in the playground with a single target file, but we plan to add support in the playground for semgrep-pro, in which case we might have to execute semgrep on multiple files.

Reply:
> plan to add support in the playground for semgrep-pro, in which case we might have to execute semgrep on multiple files.

Yea this is why, I'll add a comment

Reply:
- what exactly is `root`? is it the path we use for resolution in the case that a file is a relative path? I agree w/ Pad that we should comment this, but we're not beholden to copying ocaml's variable names and it can't hurt to make the variable more self-explanitory
- what's your plan to update semgrep-app?

Reply:
root is the root of scanning, and it's required for semgrep pro intrafile. OSS wise it doesn't do anything, it's just here to keep the interface the same

Reply:
As far as my plan to update semgrep-app, I was just going to make a PR that bumps the default version and updates the interface to use this new one. I think this should be fine since the version is hardcoded, so nothing should break in the next release